charts/ocis/values.yaml

```diff
@@ -2111,7 +2163,9 @@ services:
         # - text-editor
         # - pdf-viewer
         # - external
-        # - user-management
+        # - admin-settings
+        # - epub-reader
+        # - draw-io
```

Comment:
Is that officially available know?

Reply:
I just took it from the documentation for oCIS 6.1: https://owncloud.dev/services/web/configuration/

![image](https://github.com/user-attachments/assets/9678c973-b4bc-475c-9e42-11a54ea08b29)

Reply:
Hmm. I not sure if this is the right place to look if something is considered stable.

@micbar Can I please have your vote here? TIA

Reply:
> Hmm. I not sure if this is the right place to look if something is considered stable.

what do you mean by "stable" ? I agree that owncloud.dev might change over time because it has no concept of oCIS versions, while doc.owncloud.com has. But for rolling releases like 6.1.0 there is no documentation on doc.owncloud.com from what I know.

In this particular case and because of the timing, owncloud.dev and the released code are in sync: https://github.com/owncloud/ocis/blob/v6.1.0/services/web/pkg/config/defaults/defaultconfig.go#L101